modules/candidate_parameters/js/candidateInfo.js

```diff
@@ -32,9 +32,9 @@ var CandidateInfo = React.createClass({
       },
       success: function (data) {
         var formData = {
-          flagged_caveatemptor: data.flagged_caveatemptor,
+          flaggedCaveatemptor: data.flagged_caveatemptor,
```

Comment:
@stellarxo Lol, you broke it again. There are plenty of references that use `this.state.formData.flagged_caveatemptor`, etc